### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by Phyco-Ninja**
* removed emoji

* Update google_img.py

* Update google_reverse_search.py

* Update watch.py

* updated to spotdl v3

### CODE REVIEW

File: plugins/spotdl.py [hunk 1]
```diff
@@ -48,15 +49,19 @@ async def spotify_dl(message: Message):
...
+        cmd = f"cd {TEMP_DIR} && spotdl {song_n}"
     if cmd:
-        stdout, stderr = (await runcmd(cmd))[:2]
-        if not os.path.lexists(TEMP_DIR):
-            await message.err("Download Failed")
-            raise Exception(stdout + stderr)
-        if os.path.lexists(TEMP_DIR):
+        runn = await Term.execute(cmd)
+        while not runn.finished:
+            await asyncio.sleep(0.5)
```
Comment (rking32) [inline plugins/spotdl.py:56]: flood-waits :thinking:

File: plugins/spotdl.py [hunk 2]
```diff
@@ -48,15 +49,19 @@ async def spotify_dl(message: Message):
             return
         await message.edit("Downloading")
         quality = quality.strip()  # Just for Precautions 🤷‍♂
-        cmd = f"spotdl --song {song_n} -o {quality} -f {TEMP_DIR}"
+        cmd = f"cd {TEMP_DIR} && spotdl {song_n}"
```
Comment (rking32) [inline plugins/spotdl.py:52]: so no use of `quality` ?

File: plugins/spotdl.py (hunk 1 above)
Comment (Phyco-Ninja) [reply plugins/spotdl.py:56]: 0.5 sleep is enough. 🤔

File: plugins/spotdl.py (hunk 2 above)
Comment (Phyco-Ninja) [reply plugins/spotdl.py:52]: Spotl v3 is new and doesn't have much args in command line, https://github.com/spotDL/spotify-downloader/blob/master/spotdl/__main__.py#L43

File: plugins/spotdl.py (hunk 1 above)
Comment (rking32) [reply plugins/spotdl.py:56]: ok

File: plugins/spotdl.py (hunk 2 above)
Comment (rking32) [reply plugins/spotdl.py:52]: ok, thanks :+1:

**Pull request merged**